qb/select.go

```diff
@@ -163,3 +163,14 @@ func (b *SelectBuilder) AllowFiltering() *SelectBuilder {
 	b.allowFiltering = true
 	return b
 }
+
+// Func sets aggregation function for select clause on the query.
+func (b *SelectBuilder) Func(fn *Func) *SelectBuilder {
+	var param string
+	if len(fn.ParamNames) > 0 {
+		param = fn.ParamNames[0]
+	}
+
+	b.Columns(fn.Name + "(" + param + ")")
```

Comment:
After thinking it over it appears to me that we should have the very same thing as for insert builder here, https://github.com/scylladb/gocqlx/blob/master/qb/insert.go#L98-L114. 

`ColumnFunc` would let you call server side functions with bounded params, ColumnLit would add any literal the way you want. On top literals we can add `Count`.

Anyway, why not just use just `Columns`?

```
	B: Select("cycling.cyclist_name").Columns("MAX(stars) as max_stars").GroupBy("id"),
	S: "SELECT id,MAX(stars) as max_stars FROM cycling.cyclist_name GROUP BY id ",
```

Reply:
1. OK.
2. `Anyway, why not just use just Columns?` - I'm going to use `count(*)` quite often, so I'd prefer to have the method for it.

Reply:
@SpiritOfWill not to overdo it, how about just adding aggregate functions [1] to SelectBuilder. For example `Count` function that would append columns with appropriate cql code as LitColumn does [2]. 

WDYT?

[1] https://docs.datastax.com/en/cql/3.3/cql/cql_using/useQueryStdAggregate.html
[2] https://github.com/scylladb/gocqlx/blob/master/qb/insert.go#L99

Reply:
@mmatczuk OK. I'll just add `Count` method to `SelectBuilder`?

Reply:
For completeness it would be better to add all of them: min, max, avg, sum, and count. They are pretty similar.

Reply:
@mmatczuk Done.